hw1/eesss34690/countline.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+# Author: eesss34690
+# This is the first question for hw1.
+#
+
+# if len(sys.argv) < 2:
+#    sys.stdout.write('missing file name\n')
+if [ "$#" -lt 1 ]; then
+    echo "missing file name"
+
+#elif len(sys.argv) > 2:
+#    sys.stdout.write('only one argument is allowed\n')
+elif [ "$#" -gt 1 ]; then
+    echo "only one argument is allowed"
+
+# else:
+#    fname = sys.argv[1]
+# if os.path.exists(fname):
+#         with open(fname) as fobj:
+#             lines = fobj.readlines()
+#         sys.stdout.write('{} lines in {}\n'.format(len(lines), fname))
+else
+    fname=$1
+    if test -f "$fname"; then
+        echo "$(cat $fname | wc -l) lines in $fname"
+        
+# else:
+#         sys.stdout.write('{} not found\n'.format(fname))
+    else
+        echo "$fname not found"
+    fi
+fi
```

Comment:
See the "stop sign" here?

<img width="81" alt="Screen Shot 2021-09-15 at 14 21 15" src="https://user-images.githubusercontent.com/399122/133381061-51c60df7-f5e8-4987-bc7a-fce297c656e1.png">

This is the indication that EOL is missing.  It should be fine if you do not count the number of lines in this file.

Reply:
Sorry that I forgot to add EOF for this one. It is already on the new commit.

Reply:
No problem.  It is not wrong to omit the EOL at the end of file.  I only tried to point it out that github highlighted it that way.